paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -633,31 +630,6 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
   }
 
  private:
-  mkldnn::primitive_attr CreatePostOps(bool fuse_relu,
```

Comment:
Seems just move this to `mkldnn_reuse.h`?

Reply:
@tensor-tang Yes. Apart from moving CreatePostOps and ConvFwdPrimitiveDesc and ConvTransposeFwdPrimitiveDesc into mkldnn_reuse.h, they were also unified eg.
Two implementations of CreatePostOps were reimplemented into one and two instances of convFwdPrimitiveDesc and two instances ConvTransposeFwdPrimitiveDesc where also replaced with one function:  AcquireConvolutionPrimitiveDescriptor

Reply:
OK, I see.

So what's the difference with line 661 https://github.com/PaddlePaddle/Paddle/pull/16667/files#diff-ff3a710419e42895ac7587353a7af5bcL661.

Can this be reuse there as well ?

Reply:
@tensor-tang This is Int8 CreatePostOps it has additional params. In this PR I haven't touched int8 conv. Later on yes we could do similar reusing of functions for conv int8.

Reply:
Cool.